client/src/test/java/jp/co/soramitsu/iroha/java/Example3.java

```diff
@@ -0,0 +1,154 @@
+     * - user_b@bank, which has 0 usd
+     * </pre>
+     */
+    private static BlockOuterClass.Block getGenesisBlock() {
+        return new GenesisBlockBuilder()
+                // first transaction
+                .addTransaction(
+                        // transactions in genesis block can have no creator
+                        Transaction.builder(null)
+                                // by default peer is listening on port 10001
+                                .addPeer("0.0.0.0:10001", peerKeypair.getPublic())
+                                // create default "user" role
+                                .createRole(userRole,
+                                        Arrays.asList(
+                                                RolePermission.can_transfer,
+                                                RolePermission.can_get_my_acc_ast,
+                                                RolePermission.can_get_my_txs,
+                                                RolePermission.can_receive,
+                                                RolePermission.can_get_my_acc_txs
+                                        )
+                                )
+                                .createDomain(bankDomain, userRole)
+                                // create user A
+                                .createAccount("user_a", bankDomain, useraKeypair.getPublic())
+                                // create user B
+                                .createAccount("user_b", bankDomain, userbKeypair.getPublic())
+                                // create usd#bank with precision 2
+                                .createAsset(usdName, bankDomain, 2)
+                                // transactions in genesis block can be unsigned
+                                .build() // returns ipj model Transaction
+                                .build() // returns unsigned protobuf Transaction
+                )
+                // we want to increase user_a balance by 100 usd
+                .addTransaction(
+                        Transaction.builder(user("user_a"))
+                                .addAssetQuantity(usd, new BigDecimal("100"))
+                                .build()
+                                .build()
+                )
+                .build();
+    }
+
+    public static PeerConfig getPeerConfig() {
+        PeerConfig config = PeerConfig.builder()
+                .genesisBlock(getGenesisBlock())
+                .build();
+
+        // don't forget to add peer keypair to config
+        config.withPeerKeyPair(peerKeypair);
+
+        return config;
+    }
+
+    /**
+     * Custom facade over GRPC Query
+     */
+    public static void getTransactions(IrohaAPI api, String userId, KeyPair keyPair) {
+        // build protobuf query, sign it
+        val q = Query.builder(userId,1).getAccountTransactions(userId, 5,null, null, null, null, 1, 3).buildSigned(keyPair);
+//        make query and get it results
```

Comment:
There should be comment consistency:
```suggestion
        // build protobuf query, sign it
        val q = Query.builder(userId,1).getAccountTransactions(userId, 5,null, null, null, null, 1, 3).buildSigned(keyPair);
        // make query and get it results
```